assets/js/base/components/chip/stories/removable-chip.stories.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { Story, Meta } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { RemovableChip, RemovableChipProps } from '../removable-chip';
+
+export default {
+	title: 'WooCommerce Blocks/@base-components/RemovableChip',
```

Comment:
I'm thinking whether it makes sense to group the chips together like they were in the old Storybook implementation. I think it's probably nicer, especially if we think about how many components we are going to eventually have.

Reply:
I hear you and I initially did it that way. The reason I separated them into different files is because I think it's easier to separate concerns and scan the smaller files this way. I know it won't make things super complicated in this scenario, but if we later have a component directory that includes 5-10 subcomponents that all need stories, I think it'll be easier to find the relevant storybook entry if we follow the one component per file approach as much as possible.

Reply:
Oh, definitely agree with you there. But stories don't have to be in the same file to be grouped in the same category on the front-end. It's all a matter of naming this title path.

See [these stories](https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/trunk/assets/js/editor-components/error-placeholder/stories) and how they look on the frontend Storybook.